masonry/src/widget/sized_box.rs

```diff
@@ -266,6 +275,18 @@ impl SizedBox {
         this.ctx.request_paint_only();
     }
 
+    /// Clears padding.
+    pub fn clear_padding(this: &mut WidgetMut<'_, Self>) {
+        this.widget.padding = Insets::ZERO;
+        this.ctx.request_paint_only();
```

Comment:
Maybe? This should be a wrapper around `self.set_padding(Insets::ZERO)`.

Reply:
Fixed